docs/source/admin/ldd.rst

```diff
@@ -68,3 +68,22 @@ To load standard PDS4 data dictionary JSON file, for example, *orex_ldd_OREX_130
 run the following command::
 
   registry-manager load-dd -dd /home/pds/schema/orex_ldd_OREX_1300.JSON
+
+
+Upgrade Data Dictionary
+***********************
+
+Data dictionaries can change between major releases of the registry and/or its tools and APIs, necessitating an
+upgrade. To perform this, run the following command:
```

Comment:
Little reStructuredText trick: make the `:` at the end of this line into `::` and the next indented block gets formatted in fixed-width code font.

Reply:
Thank you. That's brilliant (although I should have seen that from the previous text)